compiler/ir/serialization.common/src/org/jetbrains/kotlin/backend/common/serialization/metadata/KlibMetadataSerializerExtension.kt

```diff
@@ -68,9 +76,28 @@ class KlibMetadataSerializerExtension(
         childSerializer: DescriptorSerializer
     ) {
         descriptorFileId(descriptor)?.let { proto.setExtension(KlibMetadataProtoBuf.functionFile, it) }
+        descriptor.findKDoc()?.let { proto.setExtension(KlibMetadataProtoBuf.functionKdoc, it.render()) }
```

Comment:
IIUC, (secondary) constructors aren't supported (here and in other parts of the machinery).